binding/dotnet/README.md

```diff
@@ -12,10 +12,21 @@ Porcupine is:
 - scalable. It can detect multiple always-listening voice commands with no added runtime footprint.
 - self-service. Developers can train custom wake word models using [Picovoice Console](https://picovoice.ai/console/).
 
+## Requirements
+
+- .NET Standard 2.0, .NET Core 3.1
```

Comment:
I know it's confusing, but because this binding targets .NET Standard 2.0 and .NET Core 3.1, it means it is compatible with applications that target .NET Core 2.0+ and .NET Framework 4.6.1+ (https://docs.microsoft.com/en-us/dotnet/standard/net-standard). You can't list .NET Standard as a requirement, because it isn't an SDK, it is just a description of a standard. I would put .NET Core SDK 3.1 as a requirement, and then list desktop platforms with compatibility of .NET Core 2.0+ and .NET Framework 4.6.1+  and Rpi, Jetson and Beaglebone with compatibility of .NET Core 3.1+